web/server/vue-cli/src/assets/userguide/userguide.md

```diff
@@ -14,429 +14,759 @@ on GitHub</a>
+column of the run list table and clicking on the `DELETE` button above the run
+list table. It will remove the run and all related data from the database.
+
+![Delete runs](images/runs/delete_runs.png)
+
+**IMPORTANT**: be careful when you delete a run because this operation can not
+be undone.
+
+## Run history
+On every run storage event a history event is created for the run. You can
+see the history events for a run if you click to the expand arrow (`v` icon)
+beside the run name column.
+
+You will see the following information for each history event:
+- `Storage date` when the run was stored to the server.
+- `User name` who performed the `CodeChecker store` operation.
+- `Show statistics` button which can be used to show the statistics for this
+history event.
+- `Check command` which was used to analyze your project. You can click to the
+console style button (`>_` icon) to see the actual check command of the
+history event.
+- `CodeChecker version` which was used during the analysis.
+- `Analyzer statistics`: information about the analysis. You can click to this
+section to see the details.
+
+![Run history](images/runs/show_run_history.png)
+
+### Compare run history events
+Calculates difference between two analyses of the same run storage, showing
+which reports have been fixed and which are newly introduced.
+
+To compare analysis history events you have to do the following:
+- select *BASELINE** runs with the first checkbox in the *Diff* column of the
+run list table for a history event.
+- select *COMPARE TO** runs with the second checkbox in the *Diff* column of
+the run list table for a history event.
+- click on the `Diff` button above the run list table.
+
+### Filter run history events
+You can filter run history events with the tag name and storage date input
+boxes above the run list table.
+
+![Filter run history events](images/runs/filter_run_history_events.png)
+
+*Note*: the filters will be saved in the URL query parameter list.
+
+# Reports
+If you select a run at the [list of runs](#runs) view, you get to this page.
+This page lists the analysis result for the given run.
+
+![Reports](images/reports/reports.png)
+
+Every report has the following fields:
+- `Report ID`
+- `Unique report identifier` (repot hash). You can find more information how
+these hashes are generated [here](https://github.com/Ericsson/codechecker/blob/master/docs/analyzer/report_identification.md).
+- `File path`: file path where the report is found.
+- `Line and column numbers` of the report where the bug is.
+- `Checker name` which found the report during the analysis.
+- `Checker message`: description about the report.
+- `Analyzer name` which found the problem during the analysis.
+- [`Severity`](#severity-levels)
+- `Bug path length`: length of the bug path. Reports with longer bug path
+are more difficult to understand.
+- [`Latest review status`](#review-status)
+- [`Latest detection status`](#detection-status)
```

Comment:
At the picture the reports are listed in uniqueing mode where these details are not visible and the detection/fix dates are only visible if I hover over the detection status icon.